viam_cartographer.go

```diff
@@ -454,9 +457,13 @@ func terminateCartoFacade(ctx context.Context, cartoSvc *CartographerService) er
 		cartoSvc.logger.Debug("terminateCartoFacade called when cartoSvc.cartofacade is nil")
 		return nil
 	}
-	stopErr := cartoSvc.cartofacade.Stop(ctx, cartoSvc.cartoFacadeTimeout)
-	if stopErr != nil {
-		cartoSvc.logger.Errorw("cartofacade stop failed", "error", stopErr)
+
+	var stopErr error
+	if !cartoSvc.cloudStoryEnabled {
+		stopErr := cartoSvc.cartofacade.Stop(ctx, cartoSvc.cartoFacadeTimeout)
```

Comment:
so `cartofacade.Start` and `cartofacade.Stop` is only used for map rate sec/saving maps?

Reply:
yep! it calls through the CGo API into C++ function [CartoFacade::Start()](https://github.com/viamrobotics/viam-cartographer/blob/e13d374dc91f6e5cc621d7c7e5ad4028feb9541c/viam-cartographer/src/carto_facade/carto_facade.cc#L681) and in that all it does is call `StartSaveInternalState();`